src/sentry/migrations/0024_auto_20191230_2052.py

```diff
@@ -34,7 +35,7 @@ def backfill_eventstream(apps, schema_editor):
     retention_days = options.get("system.event-retention-days") or DEFAULT_RETENTION
 
     def get_events(last_days):
-        to_date = datetime.now()
+        to_date = datetime.now(pytz.utc)
```

Comment:
I'd argue the more proper way to do this is through `timezone.now()` which is what we use in our codebase.

```python
from django.utils import timezone
timezone.now()
```